src/Arr.php

```diff
@@ -726,4 +726,57 @@ public function isEmpty(): bool
     {
         return $this->length() === 0;
     }
+
+    /**
+     * Returns a string representing the elements of the array
+     * @return string the string representation
+     */
+    public function toLocaleString($locale = 'en_US.utf8', $timezone = 'UTC'): string
+    {
+        if ($this->length() <= 0) {
```

Comment:
There is a method available here called `isEmpty` so we could use that to ensure that we are using the same logic to check if empty across the entire class instead of different points in the class recreating the same logic.

Reply:
Good catch